pkg/gds/store/leveldb/leveldb.go

```diff
@@ -282,7 +284,15 @@ func (s *Store) SearchVASPs(query map[string]interface{}) (vasps []*pb.VASP, err
 		log.Debug().Strs("name", names).Msg("search name query")
 		for _, name := range names {
 			if id := s.names[name]; id != "" {
+				// exact match
 				records[id] = struct{}{}
+			} else if len(name) > 2 {
```

Comment:
I think 3 is a totally reasonable number to go with and I'm glad that it's still possible to have 2-letter names (I remember hearing that this is far more common in Asia), just not < 3 and also fuzzy. If we think we might like to play around with length limits in the future, we could make this a global variable.

Reply:
I'm hoping that with 1 and 2 letter names (particularly if they are Chinese characters) prefix matching becomes less of an issue and the exact match case triggers more often. If we find ourselves in this situation we'll probably want to rethink the whole search methodology anyway - maybe Trtl/Honu can have search indices?